PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/common/ValidationDebouncedTextInput.java

```diff
@@ -58,7 +58,7 @@ public AzureValidationInfo doValidate() {
             this.setExtensions(spinner);
         } else if (info == AzureValidationInfo.OK) {
             this.setExtensions(valid);
-        } else {
+        } else if (info != AzureValidationInfo.UNINITIALIZED) {
```

Comment:
what is `AzureValidationInfo.UNINITIALIZED` used for?
it seems `AzureValidationInfo.UNINITIALIZED` is business specific, `AzureValidationInfo` is not the right place to hold it.

Reply:
it means whether a field is operated to input content by user.